init.sql

```diff
@@ -159,3 +159,202 @@ create table asset_creators
 create unique index asset_creators_asset_id on asset_creators (asset_id);
 create index asset_creator on asset_creators (asset_id, creator);
 create index asset_verified_creator on asset_creators (asset_id, verified);
+
+create type whitelist_mint_mode AS ENUM ('burn_every_time', 'never_burn');
+create type end_setting_type AS ENUM ('date', 'amount');
+
+create table candy_machine
+(
+    id                       bytea               PRIMARY KEY,
+    features                 int,
+    authority                bytea               not null,
+    mint_authority           bytea,
+    wallet                   bytea               not null,
+    token_mint               bytea,
+    items_redeemed           int                 not null,
+    candy_guard_pda          bytea,
+    version                  int                 not null,
+    collection_mint          bytea,                            
+    allow_thaw               bool,                                    
+    frozen_count             int,                                      
+    mint_start               int,
+    freeze_time              int,                                     
+    freeze_fee               int,                                      
+);
+
+create table candy_machine_data
+(
+    id                         bigserial        PRIMARY KEY,
+    uuid                       varchar(6),
+    price                      int,
+    symbol                     varchar(5)       not null,
+    seller_fee_basis_points    int              not null,
+    max_supply                 int              not null,
+    is_mutable                 bool             not null,
+    retain_authority           bool,
+    go_live_date               int,
+    items_available            int              not null,
+    candy_machine_id           bytea references candy_machine (id),
+);
+create unique index candy_machine_data_candy_machine_id on candy_machine_data (candy_machine_id);
+
+create table candy_machine_creators
```

Comment:
This seems really obvious, but I assume the vision for this is that the Vec of N creators will result in N new rows of this table, and they tie back to the candy machine all via the candy_machine_id.  That makes perfect sense to me.  Please correct me if I'm misunderstanding.

Reply:
yes and there will be an rpc method for get_candy_machines_by_creator. each one will be its own row.